drivers/ovsSwitch.go

```diff
@@ -223,6 +233,14 @@ func (sw *OvsSwitch) CreatePort(intfName string, cfgEp *OvsCfgEndpointState, pkt
 			log.Errorf("Error setting link %s up. Err: %v", ovsPortName, err)
 			return err
 		}
+
+		// Set the link mtu to 1450 to allow for 50 bytes vxlan encap
+		// (inner eth header(14) + outer IP(20) outer UDP(8) + vxlan header(8))
+		err = setLinkMtu(intfName, vxlanVethMtu)
```

Comment:
just wondering if we need to worry about the mtu of the host link itself as well?

Also should we set mtu even for internal ports?